badCode.rb

```diff
@@ -0,0 +1,5 @@
+def badName
+  if something
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.